daprdocs/content/en/operations/configuration/increase-read-buffer-size.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "How-To: Handle large http header size"
+linkTitle: "HTTP header size"
+weight: 6000
+description: "Configure a larger http read buffer size"
+---
+
+Dapr has a limit for the read buffer size which is defaults to 4 KB, this param limits the maximum http header size. Using bigger than default 4KB size headers should set this to a proper value. Otherwise, you may encounter a `Too big request header` service invocation error. You can change this config by defining `dapr.io/http-read-buffer-size` annotation or `--dapr-http-read-buffer-size` flag.
+
+
+
+{{< tabs Self-hosted Kubernetes >}}
+
+{{% codetab %}}
+
+When running in self hosted mode, use the `--dapr-http-read-buffer-size` flag to configure Dapr to use non-default request body size:
```

Comment:
```suggestion
When running in self hosted mode, use the `--dapr-http-read-buffer-size` flag to configure Dapr to use non-default http header size:
```

Reply:
I think this is the correct modification.